src/Components/Dialogs/dialogs.jsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 
 import style from './dialogs.module.css';
+import I18Y from '../../Core/constants';
```

Comment:
Consider renaming "Core" to "core".
Leading upper-case letter implies that this is a component or class.

Reply:
okey,I changed it